### ISSUE TITLE
Accessibility- All resource reports: 4.1.2- Name, Role, Value- Selected tab not announced by screen reader

### ISSUE THREAD

**Issue by aj-he** [archesproject/arches-her#1267]
**Given** I am a user reliant on a screen reader
**When** I navigate over tabs
**Then** I am only informed that the tabs are clickable and not which tab I currently have selected which is only visually signified by being underlined

Screen readers should be able to tell that a tab is currently selected or it leaves the user unsure of which tab they are currently on

**4.1.2- Name, Role, Value**
Assistive Technologies (AT) should be able to gather information about, activate(or set) and keep up to date on the status of user interface controls in the content. If custom controls are created, however, or interface elements are programmed (in code or script) to have a different role and/or function than usual, then additional measures need to be taken to ensure that the controls provide important information to assistive technologies and allow themselves to be controlled by assistive technologies. Examples include
- A particularly important state of a user interface control is whether or not it has focus. The focus state of a control can be programmatically determined, and notifications about change of focus are sent to user agents and assistive technology.
 - Other examples of user interface control state are whether or not a checkbox or radio button has been selected, or whether or not a collapsible tree or list node is expanded or collapsed.

**Comment by aj-he** [comment]
completed in #1285 and #1303
